governance-exercises/StableLab-Poll-02/Clive Derecognition as AD - August 9, 2023.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Clive Derecognition as AD - August 9, 2023
+summary: Signal your support or opposition to derecognizing Clive as an AD due to identity compromise based on the materials AD Diane provided.
+discussion_link: https://forum.makerdao.com/
+parameters:
+    input_format: single-choice
+    victory_conditions:
+        - { type : plurality }
+    result_display: single-vote-breakdown
+version: v2.0.0
+options:
+   0: Abstain
+   1: Yes
+   2: No
+start_date: 2023-08-14 16:00 UTC
+end_date: 2023-08-17 16:00 UTC
+---
+# Poll: Clive Derecognition as AD - August 9, 2023
+
+The Governance Facilitators have placed a Governance Poll into the voting system. This Governance Poll will be active for three days beginning on 8, August 2023 at 16:00 UTC.
+
+**This is a binary vote.**
+- **You may vote for a single option.**
+- **You should vote for the option which you prefer.**
+- **If you would accept either option, you should vote 'Abstain'.**
+
+## Review
+
+The community can vote in this poll to express support or opposition to the following changes:
+
+* Derecognition of Clive AD due to significant suspicion of identity discovery. According to the Atlas, if the materials provided against Clive constitute significant suspicion the aligned delegate must be derecognized.
+* AD Diane is acting as a whistle-blower by providing materials suggesting the identity of Clive. According to Atlas, Diane is eligible to receive half of the AD Buffer for providing useful information for determining that the privacy of an AD was compromised. 
+
+Please review the discussion [thread](discussion_link) to help inform your position before voting.
+
+## Outcomes
+
+**If the votes for the 'Yes' option exceed the votes for the 'No' option then the following actions will be taken:**
+
+* This change will be included in an upcoming Executive Vote.
```

Comment:
What change?

Reply:
Like, if you're talking about the bounty payout, I would not characterize that as a change to the protocol. Likewise you don't need an executive vote to derecognize Clive, so that is also not a change. 

You can't just blindly follow the templates, especially the most-generic templates.